api/specs/common/schemas/project.yaml

```diff
@@ -1,11 +1,11 @@
 components:
   schemas:
     ProjectIn:
-      $ref: "./project-v0.0.1-converted.yaml"
+      $ref: "./project-v0.0.1-pydantic-converted-clean.yaml"
```

Comment:
for me you could even just have project-v0.0.1.yaml. I am not even sure we need the version. I don't think we ever incremented it. maybe @pcrespov has some other opinion?

Reply:
What do you mean, to remove the "v0.0.1" from the name?

Reply:
both

Reply:
I would like to wait for another opinion, I like the suffix "-pydantic-converted-clean" --> it is easier to understand when you look at the Makefile what is happening there when this file is created. Regarding the version, I dont have any opinion.

Reply:
> for me you could even just have project-v0.0.1.yaml. I am not even sure we need the version. I don't think we ever incremented it. maybe @pcrespov has some other opinion?

good with me.

Reply:
i will leave the the name as it was for now